### DISCUSSION TITLE
please help

### DISCUSSION THREAD

**Discussion by mrchromebookman** [FunkinCrew/Funkin#1494] [Q&A]
when i play online and i do things it crashes please help

**Comment by MrGhost7969** [comment]
Like, anything?  Things such as navigating through the songs, playing them, etc?

**Comment by TentaRJ** [comment]
Please use the template next time when you create an issue @mrchromebookman

**Comment by exurd** [comment]
![tellustheissue](https://user-images.githubusercontent.com/7785022/112510320-3a1dce80-8d89-11eb-8701-7718f51b5aa9.gif)
